librepo/handle.h.in

```diff
@@ -170,7 +173,9 @@ typedef enum {
         Use LRO_PROXYAUTHMETHODS */
 
     LRO_PROXYUSERPWD,  /*!< (char *)
-        User and password for proxy in format user:password */
+        User and password for proxy in format user:password
+        Both the username and the password are URL decoded before use, so if the username
```

Comment:
Ah, I see, this was the behavior already, but good that it's more clearly documented now.